ton-smartcontract/src/commonMain/kotlin/org/ton/smartcontract/Text.kt

```diff
@@ -0,0 +1,32 @@
+package org.ton.smartcontract
+
+import org.ton.cell.CellBuilder
+import org.ton.cell.CellSlice
+import org.ton.tlb.*
+
+data class Text(
+    val data: SnakeData
+) {
+    companion object : TlbCodec<Text> by TextCombinator {
+        @JvmStatic
+        fun tlbCodec(): TlbCombinator<Text> = TextCombinator
+    }
+}
+
+private object TextCombinator : TlbCombinator<Text>() {
+    override val constructors: List<TlbConstructor<out Text>> by lazy { listOf(TextConstructor) }
+
+    override fun getConstructor(value: Text): TlbConstructor<out Text> = TextConstructor
+
+    private object TextConstructor : TlbConstructor<Text>(
+        schema = "text#_ {n:#} data:(SnakeData ~n) = Text;"
+    ) {
+        private val snakeDataCodec by lazy { SnakeData.tlbCodec() }
```

Comment:
remove lazy, change to `loadTlb(SnakeData)`, `storeTlb(SnakeData, value.data)`

Reply:
Snake data does not implement TlbCodec in the companion